src/server.js

```diff
@@ -48,6 +50,50 @@ module.exports = function() {
 			maxAge: 86400 * 1000,
 		}));
 
+	app.get("/uploads/:file", (req, res) => {
+		const fileName = req.params.file;
+		const uploadPath = Helper.getFileUploadPath();
+		const contentType = mime.contentType(path.extname(fileName));
+		const filePath = path.join(uploadPath, fileName);
+		const uploadedFile = upload.ensureFileExists(filePath);
+
+		// Should be browser default, but better safe and explicit rather than sorry.
+		res.setHeader("Content-Disposition", "inline");
+
+		switch (contentType) {
+		case "application/zip":
+		case "application/x-7z-compressed":
+		case "application/x-tar":
+		case "application/json":
+		case "audio/midi":
+		case "audio/mpeg":
+		case "audio/mpeg3":
+		case "audio/ogg":
+		case "audio/wav":
+		case "audio/x-mid":
+		case "audio/x-midi":
+		case "audio/x-mpeg":
+		case "audio/x-mpeg-3":
+		case "image/png":
+		case "image/gif":
+		case "image/jpg":
+		case "image/jpeg":
+		case "image/webp":
+		case "video/webm":
+		case "video/ogg":
+		case "video/mp4":
+			break;
+		default:
+			res.set("content-type", "text/plain");
```

Comment:
If i upload an `exe`, will it get printed in the browser?

Reply:
Now will be served as an attachment